### ISSUE TITLE
Ugly wall texture while in purple mushroom mode.

### ISSUE THREAD

**Issue by GoogleFrog** [SpringCabal/LD35#66]
When I turn into a purple mushroom the upper wall texture looks like smoothed pixels. This occurs in wisp mode but the blue filter makes it less obvious. I do think this this was intentional because it jars with detailed art style of the other art.
![screen00001](https://cloud.githubusercontent.com/assets/1318249/14635347/530fe5ce-0669-11e6-8a96-be1f9810dcf4.jpg)

**Comment by gajop** [comment]
Agreed. I think the top level diffuse map is generally quite poor quality. I don't think it was intentional though.

**Comment by smoth** [comment]
it isn't the texture of the wall, it is the map. I ran out of time and was unable to do my 4 splat channel because I was working on getting other assets in game. it is stuck that way unless I get clearance for further work on art assets
